AsyncDisplayKit/Details/ASDataController.h

```diff
@@ -116,6 +117,16 @@ extern NSString * const ASCollectionInvalidUpdateException;
 - (instancetype)initWithDataSource:(id<ASDataControllerSource>)dataSource eventLog:(nullable ASEventLog *)eventLog NS_DESIGNATED_INITIALIZER;
 
 /**
+ * The map that is currently displayed. The "UIKit index space."
+ */
+@property (nonatomic, strong, readonly) ASElementMap *visibleMap;
```

Comment:
True. Let's leave the name only because this property is doomed anyway – it's here right now to keep this diff small. In a follow-up diff coming soon, I'm going to have the DataController generate this and then hand it out through its delegate, rather than saving it. The data controller shouldn't really care what's "visible", it's just a processing unit at this point.